doc/filters/length.rst

```diff
@@ -9,3 +9,15 @@ the length of a string:
     {% if users|length > 10 %}
         ...
     {% endif %}
+
+.. note::
+
+    For objects implementing the ``__toString()`` magic method, the result is the
+    length of the string being returned.
+
+    If ``Countable`` is implemented as well, the return value of the ``count()`` method
+    takes precedence.
```

Comment:
I find this confusing. I prefer the way you documented the other one. Can you change it to read the same as the other one?